src/loaders/genome_collection/parse_tool_results.py

```diff
@@ -347,6 +358,115 @@ def _retrieve_pre_processed_docs(tool: str,
     return docs
 
 
+def _convert_to_iso8601(date_string: str) -> str:
+    # Convert a date string to ISO 8601 format
+    formats_to_try = ["%Y/%m/%d", "%Y-%m-%d"]  # Add more formats as needed
+
+    for date_format in formats_to_try:
+        try:
+            parsed_date = datetime.strptime(date_string, date_format)
+            iso8601_date = parsed_date.date().isoformat()
```

Comment:
Yes, for the time being. We only have two date columns, both with date precision. I anticipate that when we introduce date-time values and require additional formats, we will need to revisit and update this logic since this logic will throw an error for data-time time string.

Reply:
If you switch to dateutil it won't though - and dateutil will make date parsing a whole lot easier

Reply:
See comment above